components/BrownBanner.tsx

```diff
@@ -31,8 +31,8 @@ export const BrownBanner: React.FC<BrownBannerProps> = () => {
         size="sm"
         className={"px-2 py-6 sm:px-4"}
       >
-        <FaComments className="text-lg sm:text-xl" />
-        Work with Us
+        <FaHandshake className="text-lg sm:text-2xl" />
+        Work With Us
```

Comment:
I liked the previous casing.. but am I wrong??? I have no idea. I'm reading this [blog post](https://titlecaseconverter.com/blog/is-with-capitalized/) and now i don't know

```suggestion
        Work with Us
```

Reply:
I kind of like it capitalized but I don't really feel strongly about it. Seems like we can go either way according to that post.